tests/integration/dataframe/test_dataframe.py

```diff
@@ -105,6 +121,26 @@ def supports_test_result_id(client: DataFrameClient) -> bool:
     return result
 
 
+@pytest.fixture(scope="class")
+def table_with_data(client: DataFrameClient, create_table) -> str:
```

Comment:
Note: I further adjusted the tests to only create a table with data once and reuse that table across multiple tests. That way we don't have to wait for the data to be available multiple times.